### PR TITLE
Databases week3

### PR DESCRIPTION
**Pull request by RemyaManoharan**

### CODE REVIEW

File: databases/week3/mealsharing_ddl.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+CREATE TABLE `tbl_review` (
+  `review_id` int(10) NOT NULL AUTO_INCREMENT,
+  `meal_id` int(10) NOT NULL,
+ `review_date` datetime NOT NULL,
+  `rating` int(5) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  PRIMARY KEY (`review_id`),
+  CONSTRAINT `fk_tbl_review_meal` FOREIGN KEY (`meal_id`) REFERENCES `tbl_meals` (`meal_id`) ON DELETE CASCADE
+  
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```
Comment (Elia-Git) [inline databases/week3/mealsharing_ddl.sql]: Cool. Please be ready to remember the table names or you can rename them as they were given (`meal, review & reservation` ) as we are going to use them in the coming modules.

File: databases/week3/mealsharing_dml.sql [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ORDER BY created_date DESC;    
+-- Sort all meals by average number of stars in the reviews
+SELECT 
+    m.title, AVG(rating) AS avg_rating
+FROM
+    tbl_meals m
+        RIGHT JOIN
+    tbl_review rv ON m.meal_id = rv.meal_id
+GROUP BY m.title
+ORDER BY AVG(rating) DESC;
```
Comment (Elia-Git) [inline databases/week3/mealsharing_dml.sql]: Where clause, is better used to sort based on a unique key, id in this case & wondering to know why you use right join here. Column rating was supposed to star in the review table review if we have to make it as per the given source of the hw.

**Review by Elia-Git — APPROVED** [review]
Good job. 
- Better to align the table names as given in the hw.
- It is generally recommended to name tables in the singular form rather than the plural form.
 so pls create table names in a singular form.

**Comment by RemyaManoharan** [comment]
Thank Elias for the feedback was worth. I have changed accordingly

**Comment by RemyaManoharan** [comment]
Changes done

**Pull request merged**